realm/realm-library/src/main/cpp/CMakeLists.txt

```diff
@@ -157,8 +152,8 @@ endif()
 #     -Wno-missing-field-initializers disable in object store as well.
 set(WARNING_CXX_FLAGS "-Werror -Wall -Wextra -pedantic -Wmissing-declarations \
     -Wempty-body -Wparentheses -Wunknown-pragmas -Wunreachable-code \
-    -Wno-missing-field-initializers  -Wno-maybe-uninitialized -Wno-uninitialized")
-set(REALM_COMMON_CXX_FLAGS "${REALM_COMMON_CXX_FLAGS} -DREALM_ANDROID -DREALM_HAVE_CONFIG -DPIC -pthread -fvisibility=hidden -std=c++14 -fsigned-char")
+    -Wno-missing-field-initializers -Wno-uninitialized")
+set(REALM_COMMON_CXX_FLAGS "${REALM_COMMON_CXX_FLAGS} -DREALM_ANDROID -DREALM_HAVE_CONFIG -DPIC -pthread -fPIC -frtti -fvisibility=hidden -std=c++14 -fsigned-char")
```

Comment:
It is strange there is a `-std=c++11` in https://github.com/realm/realm-java/pull/6460#issuecomment-493427068 before `-std=c+=14`.

maybe we should use the cmake way to set the std version. See https://cmake.org/cmake/help/v3.1/prop_tgt/CXX_STANDARD.html

Reply:
That is just the way variables from different places are added, the latest wins, so shouldn't be a problem